src/main/java/org/stellar/base/Memo.java

```diff
@@ -0,0 +1,81 @@
+package org.stellar.base;
+
+import org.apache.commons.codec.DecoderException;
+import org.stellar.base.xdr.MemoType;
+import org.stellar.base.xdr.Uint64;
+
+import org.apache.commons.codec.binary.Hex;
+
+import java.io.UnsupportedEncodingException;
+
+public class Memo {
+    public static org.stellar.base.xdr.Memo none() {
+        org.stellar.base.xdr.Memo memo = new org.stellar.base.xdr.Memo();
+        memo.setDiscriminant(MemoType.MEMO_NONE);
+        return memo;
+    }
+
+    public static org.stellar.base.xdr.Memo text(String text) {
+        try {
+            if (text.getBytes("UTF8").length > 28) {
+                throw new RuntimeException("text must be <= 28 bytes.");
+            }
+        } catch (UnsupportedEncodingException e) {
+            return null;
```

Comment:
I'm not comfortable hiding this exception.  We should be bubbling it up instead of failing silently.